client/cli/go/cmds/topology.go

```diff
@@ -126,6 +126,9 @@ var topologyLoadCommand = &cobra.Command{
 
 		// Load current topolgy
 		heketiTopology, err := heketi.TopologyInfo()
+		if err != nil {
+			return err
```

Comment:
Please reword return a nice error and append err to it.  For example:

```go
return fmt.Errorf("Unable to get topology information: %v\n", err)
```

Also, there seems to be a go fmt error.  Please fix